packages/gcp/data_stream/firestore/_dev/deploy/tf/main.tf

```
@@ -0,0 +1,33 @@
+variable "TEST_RUN_ID" {
+  default = "detached"
+}
+
+variable "project_id" {
```

This has been changed and should be:
```suggestion
variable "gcp_project_id" {
```

---

I noticed this change was not included in https://github.com/elastic/integrations/pull/2704/commits/081c2ce70dfd0a67e2e8eeb79c3dfe3dc8eab89e, is that done on purpose?